config/locales/nl.yml

```diff
@@ -0,0 +1,210 @@
+---
+nl:
+  activerecord:
+    errors:
```

Comment:
Dit is de standaard `nl.yml` i18n file, daar ga ik verder niet in lopen knippen en plakken.